docs/status.md

```diff
@@ -88,7 +88,7 @@ one of the following tracking labels.
 | dynamic_update_slice     | no            | revisit      | no             | yes             | no          |
 | einsum                   | no            | revisit      | no             | no              | no          |
 | exponential              | yes           | yes          | yes            | yes             | no          |
-| exponential_minus_one    | no            | yes*         | yes*           | yes             | no          |
+| exponential_minus_one    | yes           | revisit      | yes            | yes             | no          |
```

Comment:
Can you mention the verification ticket for revisit

Reply:
Sure, it's mentioned in a previous [comment](https://github.com/openxla/stablehlo/pull/502#discussion_r1019756530), but I'll mention it as a standalone comment for visibility.